nb-configuration.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IDE specific config files should not be committed

Reply:
Unwanted files removed from project. They added to .gitignore.